src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,242 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
+
+        private T element;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(T element, Node<T> next, Node<T> prev) {
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor, which you are doing correctly. However, the getters and setters for `Node` are not needed since it's an inner class and we have access to its private fields.

Reply:
I have already done it